src/plugins/discover/public/utils/get_sort.ts

```diff
@@ -71,3 +74,22 @@ export function getSortArray(sort: SortPair[], dataView: DataView): SortPairArr[
     return acc;
   }, []);
 }
+
+/**
+ * sorting for embeddable, like getSortArray,but returning a default in the case the given sort or dataView is not valid
+ */
+export function getSortForEmbeddable(
+  sort?: SortPair[],
```

Comment:
```suggestion
  sort?: SortInput,
```

Can't it be in both formats here: old and new?

Reply:
True, thx!